From 8009b80450143c439fcd940c1f5fa4173013d9b5 Mon Sep 17 00:00:00 2001
From: Nathan Owens <ndowens@artixlinux.org>
Date: Sun, 27 Dec 2020 13:11:09 -0600
Subject: [PATCH] arm

config script detects host CPU and enables x86 flags
ï¿¼which are incompatible with ARM*
---
 wscript | 12 ------------
 1 file changed, 12 deletions(-)

diff --git wscript wscript
index 291e234..27d45e9 100644
--- wscript
+++ wscript
@@ -531,18 +531,6 @@ int main() { return 0; }''',
             # determine processor flags via /proc/cpuinfo
             #
 
-            if conf.env['build_target'] != 'i386':
-
-                flag_line = os.popen ("cat /proc/cpuinfo | grep '^flags'").read()[:-1]
-                x86_flags = flag_line.split (": ")[1:][0].split ()
-
-                if "mmx" in x86_flags:
-                    compiler_flags.append ("-mmmx")
-                if "sse" in x86_flags:
-                    build_host_supports_sse = True
-                if "3dnow" in x86_flags:
-                    compiler_flags.append ("-m3dnow")
-
             if cpu == "i586":
                 compiler_flags.append ("-march=i586")
             elif cpu == "i686":
-- 
2.29.2

