From 3642241d7c77a6fee8068cf81dc44a604fb4216a Mon Sep 17 00:00:00 2001
From: Nathan Owens <ndowens@artixlinux.org>
Date: Mon, 16 Nov 2020 15:43:09 -0600
Subject: [PATCH] musl-cdefs

---
 libelf/elf.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git libelf/elf.h libelf/elf.h
index 6439c1a..23cea49 100644
--- libelf/elf.h
+++ libelf/elf.h
@@ -19,6 +19,16 @@
 #ifndef _ELF_H
 #define	_ELF_H 1
 
+#if !defined(__BEGIN_DECLS)
+/* C++ needs to know that types and declarations are C, not C++.  */
+#ifdef  __cplusplus
+# define __BEGIN_DECLS  extern "C" {
+# define __END_DECLS    }
+#else
+# define __BEGIN_DECLS
+# define __END_DECLS
+#endif
+#endif
 /* Standard ELF types.  */
 
 #include <stdint.h>
-- 
2.29.2

